azure/azuredeploy.json

```diff
@@ -205,7 +205,7 @@
                             "sourceAddressPrefix": "*",
                             "destinationAddressPrefix": "*",
                             "access": "Allow",
-                            "priority": 101,
+                            "priority": 102,
```

Comment:
Couldn't quite follow this change :) Can you note the reason behind changing the priority here and in other places ?

Reply:
The old priorities conflict with some priorities from default network security policies that Azure deploys. The conflicts then result to failed deployments. I got these numbers by trial and error until there were no more conflicts in deployments. Depending on how the default Azure policies change these priorities might need to be changed as well in the future so I am not super happy with this hacky approach I took here.